views/layout.html

```diff
@@ -25,6 +25,9 @@
       <script type="text/javascript" src="{{assetPath}}/js/bundle.js"></script>
     </main>
   {{/main}}
+  {{$cookieMessage}}
+    <p>GOV.UK uses cookies to make the site simpler. <a href="https://www.gov.uk/help/cookies">Find out more about cookies</a></p>
```

Comment:
Can we add the content to the translations https://github.com/UKHomeOfficeForms/hof-template-partials/tree/master/translations/src/en